website/source/api/secret/kv/versioned-kv.html.md

```diff
@@ -18,13 +18,56 @@ This documentation assumes the kv secrets engine is enabled at the
 possible to enable secrets engines at any location, please update your API calls
 accordingly.
 
+
+## Configure the Versioned K/V Backend
+
+This path configures backend level settings that are applied to every key in the
+key-value store.
+
+| Method   | Path                         | Produces               |
+| :------- | :--------------------------- | :--------------------- |
+| `POST`   | `/secret/config`             | `204 (empty body)`     |
+| `PUT`    | `/secret/config`             | `204 (empty body)`     |
```

Comment:
I believe POST is all that is required to display.

Reply:
Yeah, the rest of the docs use POST only.

Reply:
This was copied from the existing k/v docs, i'll remove it in both